ForestSubmission/20241121_ForestSubmissionGtoCCbar2018/forest_miniAOD_run2_DATA.py

```diff
@@ -0,0 +1,285 @@
+
+# root output
+process.TFileService = cms.Service("TFileService",
+    fileName = cms.string("HiForestMiniAOD.root"))
+
+# # edm output for debugging purposes
+# process.output = cms.OutputModule(
+#     "PoolOutputModule",
+#     fileName = cms.untracked.string('HiForestEDM.root'),
+#     outputCommands = cms.untracked.vstring(
+#         'keep *',
+#         )
+#     )
+
+# process.output_path = cms.EndPath(process.output)
+
+###############################################################################
+
+# event analysis
+process.load('HeavyIonsAnalysis.EventAnalysis.hltanalysis_cfi')
+process.load('HeavyIonsAnalysis.EventAnalysis.hievtanalyzer_data_cfi')
+process.load('HeavyIonsAnalysis.EventAnalysis.skimanalysis_cfi')
+process.load('HeavyIonsAnalysis.EventAnalysis.hltobject_cfi')
+process.load('HeavyIonsAnalysis.EventAnalysis.l1object_cfi')
+
+from HeavyIonsAnalysis.EventAnalysis.hltobject_cfi import trigger_list_data
+process.hltobject.triggerNames = trigger_list_data
+
+process.load('HeavyIonsAnalysis.EventAnalysis.particleFlowAnalyser_cfi')
+
+process.particleFlowAnalyser.ptMin = 0
+process.particleFlowAnalyser.absEtaMax = 7
+
+################################
+# electrons, photons, muons
+SSHIRun2018A = "HeavyIonsAnalysis/EGMAnalysis/data/SSHIRun2018A.dat"
+process.load('HeavyIonsAnalysis.EGMAnalysis.correctedElectronProducer_cfi')
+process.correctedElectrons.correctionFile = SSHIRun2018A
+
+process.load('HeavyIonsAnalysis.MuonAnalysis.unpackedMuons_cfi')
+#process.load("HeavyIonsAnalysis.MuonAnalysis.muonAnalyzer_cfi")
+process.load("HeavyIonsAnalysis.MuonAnalysis.hltMuTree_cfi")
+process.load('HeavyIonsAnalysis.EGMAnalysis.ggHiNtuplizer_cfi')
+process.ggHiNtuplizer.doMuons = cms.bool(False)
+process.ggHiNtuplizer.doElectrons = cms.bool(True)
+process.ggHiNtuplizer.electronSrc = "correctedElectrons"
+process.load("TrackingTools.TransientTrack.TransientTrackBuilder_cfi")
+################################
+# jet reco sequence
+process.load('HeavyIonsAnalysis.JetAnalysis.akCs4PFJetSequence_pponPbPb_data_cff')
+################################
+# tracks
+process.load("HeavyIonsAnalysis.TrackAnalysis.TrackAnalyzers_cff")
+###############################################################################
+
+# ZDC RecHit Producer
+process.load('HeavyIonsAnalysis.ZDCAnalysis.QWZDC2018Producer_cfi')
+process.load('HeavyIonsAnalysis.ZDCAnalysis.QWZDC2018RecHit_cfi')
+
+process.load('HeavyIonsAnalysis.ZDCAnalysis.zdcanalyzer_cfi')
+process.zdcanalyzer.doZDCRecHit = True
+process.zdcanalyzer.doZDCDigi = False
+process.zdcanalyzer.zdcRecHitSrc = cms.InputTag("QWzdcreco")
+process.zdcanalyzer.calZDCDigi = True
+################################
+
+
+process.Trigger = cms.EDFilter( "TriggerResultsFilter",
+      triggerConditions = cms.vstring(
+         #'HLT_HIEle*',
+         #'HLT_HIDoubleEle*',
+         'HLT_HIL3Mu12_*',
+         'HLT_HIL3Mu15_*',
+         'HLT_HIL3Mu20_*',
+         'HLT_HIL3DoubleMu*',
```

Comment:
@Cristian-Baldenegro could you check with Florian to see which one of these three HLT paths we should be using?

Reply:
Checked with Florian, we can use `HLT_HIL3DoubleMuOpen` for this analysis.

Reply:
OK, does he know what are HLT_HIL1DoubleMuOpen and HLT_HIL2DoubleMuOpen?